src/main/resources/config/liquibase/changelog/20240626200000_changelog.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet id="20240626200000_1" author="bbesrour">
+        <!-- Create the new build_job_config table -->
+        <createTable tableName="programming_exercise_build_config">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="sequential_test_runs" type="BOOLEAN"/>
```

Comment:
Can we add default values here and make them non-nullable? At least for the boolean and int columns

Reply:
I added default values for columns that had them originally.